boards/nxp/mimxrt1170_evk/mimxrt1170_evk-pinctrl.dtsi

```diff
@@ -229,6 +229,17 @@
 		};
 	};
 
+	pinmux_lpuart2_flowcontrol: pinmux_lpuart2_flowcontrol {
```

Comment:
Please update the documentation and maybe add a `Bluetooth` section and indicate the command to fetch the blobs.

Reply:
Updated.